.stackblitz/nuxt.config.ts

```diff
@@ -0,0 +1,7 @@
+// https://v3.nuxtjs.org/api/configuration/nuxt.config
+export default defineNuxtConfig({
+  modules: ['@nuxtjs/cloudinary'],
+  cloudinary: {
+    cloudName: 'ddtc5sowb'
```

Comment:
i personally would recommend making this an environment variable so someone who's spinning this up would need to use their own account, unless this stackblitz has another purpose

other future features like uploading, which requires the extra key/secret would conflict with this

Reply:
The idea with the stackblitz link is to get the user starting as soon as possible. This is just a demo with the cloudName that belongs to my free account.

When using the module in users nuxt application they will be using environment variables as indicated in the docs :)

Reply:
yeah - the only issue you may run into is people uploading to your account at that point - so if its not already, maybe you could at least make it a separate free account so worst case, it wont impact YOUR account and it wont ever cost you money, but maybe thats already what you're doing